lib/sensors/mux.hpp

```diff
@@ -3,37 +3,31 @@
 #include "i2c_sensors.hpp"
 
 #include <array>
-#include <cstdint>
-#include <cstdio>
 #include <memory>
-#include <optional>
 
-#include <core/logger.hpp>
 #include <io/i2c.hpp>
 
 namespace hyped::sensors {
 
-static constexpr uint8_t kDefaultMuxAddress = 0x70;
-
-enum class MuxOperationResult { kFailure = 0, kSuccess };
+static constexpr std::uint8_t kDefaultMuxAddress = 0x70;
 
 template<class T, std::size_t size>
```

Comment:
Let's stick with the standard library and call it `std::size_t N`; cf. https://en.cppreference.com/w/cpp/container/array.